src/index.html

```diff
@@ -4,14 +4,176 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input 
+          type="text" 
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="surname"
            id="surname"
            autocomplete="off"
          >
```
Formatting not fixed. Please fix all cases
![image](https://user-images.githubusercontent.com/96409563/223673488-dff5f501-bcb7-427f-8f3e-2e685ac5fdae.png)